test/fixtures/dialects/postgres/postgres_select.sql

```diff
@@ -59,3 +59,5 @@ SELECT a < b COLLATE "de_DE" FROM test1;
 SELECT a COLLATE "de_DE" < b FROM test1;
 
 SELECT * FROM test1 ORDER BY a || b COLLATE "fr_FR";
+
+SELECT FROM test1;
```

Comment:
What's this test for?

Reply:
To test select clause elements are optional in Postgres (added in this change). Note it's `SELECT FROM` and not `SELECT * FROM`.